python/tox.ini

```diff
@@ -77,10 +77,10 @@ commands_pre =
   anthropic-latest: uv pip install -U anthropic
 commands =
   ruff: ruff format {posargs:.}
-  ruff: ruff check {posargs:.} --fix
+  ruff: ruff check --fix {posargs:.}
   mypy: mypy {posargs:.}
-  test: pytest -n auto {posargs:tests}
-  ci: ruff format . --diff
-  ci: ruff check . --no-fix
-  ci: mypy .
-  ci: pytest -n auto tests
+  test: pytest -n auto --nf {posargs:.}
+  ci: ruff format --diff {posargs:.}
+  ci: ruff check --no-fix {posargs:.}
+  ci: mypy {posargs:.}
+  ci: pytest -n auto -x -ra {posargs:.}
```

Comment:
What does postargs do? does it let you add your own args afterwards?

Reply:
yes, it adds a bit more flexibility if you want to say type-check or lint a specific sub folder